SceneMetadata/Editor/MetadataAssetModificationProcessor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using UnityEditor;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Unity.Labs.SuperScience
+{
+    class MetadataAssetModificationProcessor : UnityEditor.AssetModificationProcessor
+    {
+        const string k_ExampleScenePath = "SuperScience/SceneMetadata/ExampleScene.unity";
```

Comment:
Love the new use-case. Can we change this to maybe use an Asset label or some other way to mark the scene, rather than a hard-coded path?

Reply:
I considered using an asset label but I thought that would add more noise to this example than just checking a hard-coded path.

Reply:
Yeah, but hard-coded paths aren't a best practice. I wouldn't want users to think this was the right way to do it.

Reply:
Done